systems/controllers/CMakeLists.txt

```diff
@@ -41,13 +47,12 @@ if (MATLAB_FOUND)
       pods_use_pkg_config_packages(instantaneousQPControllermex gurobi)
 
       add_mex(constructQPDataPointerMex constructQPDataPointerMex.cpp)
-      target_link_libraries(constructQPDataPointerMex drakeQP drakeControlUtil)
-      pods_use_pkg_config_packages(constructQPDataPointerMex gurobi)
+      target_link_libraries(constructQPDataPointerMex drakeQP drakeControlUtil drakeQPCommon)
+      pods_use_pkg_config_packages(constructQPDataPointerMex gurobi drake-qp-common)
```

Comment:
this was part of an attempt on my part to get the drake c++ lcm headers to work. I've removed it